Tests/CurrencyFormatterServiceTest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+require_once 'src/Services/CurrencyFormatterService.php';
+use PHPUnit\Framework\TestCase;
+
+class CurrencyFormatterServiceTest extends TestCase
+{
+    public function testGetCurrencySuccess()
```

Comment:
think your missing some tests. what about an int? what about a float with 2 decimal places? what about a float with more than 2 decimal places? what about other currencies? what about malformed tests for the other params? what about failure tests for the other params?